apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/LoggingSnippets.cs

```diff
@@ -211,6 +264,43 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         // End sample
     }
 
+    /// <summary>
+    /// A simple web application to test the <see cref="GoogleLogger"/>
+    /// and associated classes. Configure <see cref="GoogleLogger"/> to
+    /// write the URL where logs can be found.
+    /// </summary>
+    internal class LoggingTestApplicationWriteUrl
+    {
+        private static readonly string ProjectId = TestEnvironment.GetTestProjectId();
+        internal static readonly TextWriter textWriter = new StringWriter();
```

Comment:
Could we make this a property (perhaps just `Writer`?) rather than a field? I know it's only for tests, but non-private fields simply make me twitchy :)

Reply:
Secondly, this being static makes me nervous. I suspect if we have multiple tests, they could give false positives. (I don't know enough about ASP.NET Core infrastructure to see immediately how to avoid that, but I suspect there's a way.)